src/Helper/CallbackQuery.php

```diff
@@ -16,7 +14,6 @@ public function __construct(
     /**
      * Execute callback function and return stats (number of inserts and updates).
      */
-    #[ArrayShape(['insert' => 'int|mixed', 'update' => 'int|mixed', 'delete' => 'int|mixed', 'log' => 'array'])]
```

Comment:
Te varētu aizvietot ar PHPStan alternatīvu.
No otras puses, visu statistikas uzskaiti vajadzētu refaktorēt ar "Telemetry" servisu vai tml.

Reply:
ieliku phpstan, tikai bļauj par to log lauku, kas ir array bez tipa